packages/pwa/app/translations/README.md

```diff
@@ -69,30 +71,46 @@ For example, in your React component, you can add formatted messages like `intl.
 
 At the minimum, only defaultMessage is the required parameter. The message id is optional. If you don’t specify it, the id is auto-generated for you.
 
-### Workflow Between Developers and Translation Team
+## Workflow Between Developers and Translation Team
 
 During your development, you extract and compile the messages and pass the generated JSON files to the translation team, as well as using the translated files in the app.
 
 We’ve provided a few npm scripts to help you:
 
--   `npm run extract-messages` to extract the default messages
+-   `npm run extract-default-translations` to extract the default messages
     -   so you don’t have to create the JSON file yourself by hand
--   `npm run compile-messages` to compile all locales into Abstract Syntax Tree (AST) format
+-   `npm run compile-translations` to compile all locales into Abstract Syntax Tree (AST) format
     -   `react-intl` can use this AST format to bypass the parsing step and thus, save a bit of time.
+-   `npm run build-translations` to run both of the above
+    -   run this whenever you add a new formatted message to your components to update the default locale's translations
 
 Whenever you do a build, the extract and compile scripts are run automatically in case you forgot to extract the latest messages.
 
-### Testing with a Pseudo Locale
+## Testing with a Pseudo Locale
 
 To check whether you’ve wrapped all of the hardcoded strings with either `intl.formatMessage()` or `<FormattedMessage />` , there’s a quick way to test that by running `npm run start:pseudolocale`. It runs your local dev server with the locale forced to the pseudo locale.
 
 Loading the site in your browser, you can quickly see that those messages that have been formatted have a long `SSS` appended to them.
 
-### Tips
+## Summary
+
+Putting all of the above together, the process for adding a new locale is as follows:
+
+1. Enable the new locale in Business Manager of your B2C Commerce instance
+2. Add the new locale to locale.config.js
```

Comment:
Sorry @vcua-mobify we can no longer assume that localization.config.js is there, as I have just closed my PR #152 . See https://github.com/SalesforceCommerceCloud/pwa-kit/pull/152#issuecomment-935138511 for more details.

For your PR, can you add an additional step to this readme here? To mention about updating the `defaultLocaleMessages` too in `App`.

Afterwards, once we have the better implementation, we can go back and update this readme.

Reply:
@vmarta The readme has been updated to remove references to locale.config.js